cypress/support/commands.js

```diff
@@ -48,4 +48,17 @@ Cypress.on('uncaught:exception', (err, runnable) => {
            });
      }
     });
-  });
\ No newline at end of file
+  });
+  Cypress.Commands.add("typeAndEnter", (selector, text) => {
+    cy.get(selector).should('be.visible').type(text);
+    cy.get(selector).type('{enter}');
+    
+  });
+  Cypress.Commands.add('checkSearchResults', (expectedResultCount, searchTerm) => {
+    cy.get('span.gf-summary b').should('have.text', expectedResultCount).then((resultCount) => {
```

Comment:
I think selector can change for the different search bars. Also, again counting the returned search element is not a good practice as it will make your test fluky.